### ISSUE TITLE
ModuleNotFoundError: No module named 'shap.utils'

### ISSUE THREAD

**Issue by gracecarrillo** [shap/shap#1355]
After `!pip install git+https://github.com/slundberg/shap.git` with today's version, I get:

```
/usr/local/lib/python3.6/dist-packages/statsmodels/tools/_testing.py:19: FutureWarning: pandas.util.testing is deprecated. Use the functions in the public API at pandas.testing instead.
  import pandas.util.testing as tm
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-7-d1d468588b4d> in <module>()
     23 from xgboost import plot_importance
     24 from sklearn.metrics import explained_variance_score, mean_squared_error, r2_score
---> 25 import shap
     26 
     27 # set plot preferences

2 frames
/usr/local/lib/python3.6/dist-packages/shap/explainers/_permutation.py in <module>()
----> 1 from ..utils import partition_tree_shuffle, MaskedModel
      2 from .._explanation import Explanation
      3 from ._explainer import Explainer
      4 import numpy as np
      5 import pandas as pd

ModuleNotFoundError: No module named 'shap.utils'
```

I'm on Google Colab Pro.

**Comment by ManuelZ** [comment]
I'm getting the same error. Could anybody point me out to a recent commit that works?

**Comment by ManuelZ** [comment]
This  commit worked for me: f3369c7
Although it comes before the latest API change, but what I wanted was the categorical variables support of issue 373, so it's good for me.
Thanks for this awesome package!

`python -m pip install https://github.com/slundberg/shap/archive/f3369c7.zip`

**Comment by harshit-bajpai** [comment]
I can recreate the error as well. It is with the latest commit v0.36.0.

**Comment by milah** [comment]
I edited the shap/setup.py this [line](https://github.com/slundberg/shap/blob/master/setup.py#L100)
`        packages=[
            'shap', 'shap.explainers', 'shap.explainers.other', 'shap.explainers._deep',
            'shap.plots.colors', 'shap.plots', 'shap.benchmark', 'shap.maskers', 'shap.utils'
        ],
`

**Comment by harshit-bajpai** [comment]
> I edited the shap/setup.py this [line](https://github.com/slundberg/shap/blob/master/setup.py#L100)
> ` packages=[ 'shap', 'shap.explainers', 'shap.explainers.other', 'shap.explainers._deep', 'shap.plots.colors', 'shap.plots', 'shap.benchmark', 'shap.maskers', 'shap.utils' ],`

This works for me. Thanks, @milah

**Comment by gracecarrillo** [comment]
The fix has been merged to master in commit fea1918c008b248d88f77d66b951504a374badcd. The latest `pip install` version works fine now.
